app/views/animals/show.html.erb

```diff
@@ -5,9 +5,13 @@
     <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Corporis odio sed, aliquid magni, porro consequatur dolorum voluptatem laborum autem ipsam.</p>
     <div class="card" style="width: 24rem;">
       <div class="card-body">
-        <h5 class="card-title">Card title</h5>
-        <p class="card-text">Some quick example text to build on the card title and make up the bulk of the card's content.</p>
-        <a href="#" class="btn btn-primary">Reserve</a>
+        <%= simple_form_for [@animal, @booking] do |f| %>
+          <%= f.input :date %>
```

Comment:
The date is not displaying for me on the dropdown, I can't seem to select one.  Is it too narrow?